src/Components/UI/VerticalNavigation/VerticalNavigation.jsx

```diff
@@ -0,0 +1,72 @@
+import { NavLink } from "react-router-dom";
```

Comment:
Appreciation the UI for the vertical navigation looks amazing with that consistent bg blur. Good work.